agr_api/src/main/java/org/alliancegenome/api/service/ExpressionRibbonESService.java

```diff
@@ -120,8 +120,16 @@ private RibbonEntity getExpressionRibbonSummary(String geneID) {
 		Pagination pagination = new Pagination(1, 250000, null, null);
 		JsonResultResponse<GeneExpressionDocument> expressionAnnotations = expressionService.getExpressionAnnotations(List.of(geneID), null, null, pagination);
 
-		Gene gene = expressionAnnotations.getResults().get(0).getGeneExpressionAnnotation().getExpressionAnnotationSubject();
-		String dataProvider = expressionAnnotations.getResults().get(0).getGeneExpressionAnnotation().getDataProvider().getAbbreviation();
+		String dataProvider;
+		Gene gene;
+		if (CollectionUtils.isNotEmpty(expressionAnnotations.getResults())) {
+			gene = expressionAnnotations.getResults().get(0).getGeneExpressionAnnotation().getExpressionAnnotationSubject();
+			dataProvider = expressionAnnotations.getResults().get(0).getGeneExpressionAnnotation().getDataProvider().getAbbreviation();
+		} else {
+			gene = new Gene();
+			dataProvider = null;
```

Comment:
combine this assignment with line 123
String dataProvider = null;
then you don't need another dataProvider assignment in the else clause.
Same with the gene assignment. You can get rid of the else clause all together.